index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>setup and mobile first</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <main>
+      <div class="illustration-head-line">
+        <!-- Starting Tool bar-->
+        <nav class="nav">
+          <div>
+            <h2 class="Subhead Neutral-1">Welcome</h2>

```

Comment:
**OPTIONAL**- To help improve your site's HTML structure, it's advisable that your `hx` tags follow an ascending hierarchy, i.e use an `h2` tag after an `h1` tag, an `h3` tag should be used after an `h2` tag. Please fix this, I advise using a `p` tag for this part 😄